yapapi/rest/configuration.py

```diff
@@ -44,27 +42,27 @@ def app_key(self) -> str:
 
     @property
     def market_url(self) -> str:
-        """The URL of the Market REST API"""
+        """Return URL of the Market REST API."""
```

Comment:
and again, in case of properties, I believe the docstrings should be formed just as they would be for attributes rather than functions...